sample/package.json

```diff
@@ -16,6 +16,7 @@
     "leaflet": "1.0.0-beta.2.rc.2"
   },
   "devDependencies": {
+    "karma": "^1.3.0",
```

Comment:
I don't see any call to karma in the PR; was this a change forgotten from a previous commit?